fusor-ember-cli/app/components/cancel-back-next.js

```diff
@@ -11,16 +11,17 @@ export default Ember.Component.extend({
 
     saveAndCancelDeployment() {
       this.get('targetObject').send('saveAndCancelDeployment');
-      this.set('openModal', false);
+      return this.set('openModal', false);
```

Comment:
@isratrade do you have thoughts on @cfchase comment about the 'return' in the action methods?